flask_security/core.py

```diff
@@ -88,13 +89,14 @@
     'CHANGE_SALT': 'change-salt',
     'REMEMBER_SALT': 'remember-salt',
     'DEFAULT_REMEMBER_ME': False,
-    'DEFAULT_HTTP_AUTH_REALM': 'Login Required',
-    'EMAIL_SUBJECT_REGISTER': 'Welcome',
-    'EMAIL_SUBJECT_CONFIRM': 'Please confirm your email',
-    'EMAIL_SUBJECT_PASSWORDLESS': 'Login instructions',
-    'EMAIL_SUBJECT_PASSWORD_NOTICE': 'Your password has been reset',
-    'EMAIL_SUBJECT_PASSWORD_CHANGE_NOTICE': 'Your password has been changed',
-    'EMAIL_SUBJECT_PASSWORD_RESET': 'Password reset instructions',
+    'DEFAULT_HTTP_AUTH_REALM': _('Login Required'),
```

Comment:
Maybe, I should have elaborate a bit more at the beginning. There are two steps for making the extension configurable and translation friendly. 

1. Make sure that all strings are properly extracted by Babel (usually done by `_` or `*gettext` function).
2. Render translated string in UI.

For the first step we need to make sure that default configuration options are marked. This can be simply done by an identity function (`def _(x): return x`). In order to make second step correctly we need to make sure that the `gettext` (or it's lazy equivalent) function is call on each string that a user can see.

**What if developer provides different configuration for Flask-Security messages?**
First, (s)he needs to make sure that these strings are extracted from the sources and included in `flask_security.mo` that it's loaded by Flask-BabelEx.

I hope I have explained the problems well.